coralproject/talk
# [CORL-977] Allow changing default story mode

## What does this PR do?

Allows switching default story mode to Q&A based on `DEFAULT_QA_STORY_MODE` feature flag.



## What changes to the GraphQL/Database Schema does this PR introduce?



Adds a new `enum` value to the `FEATURE_FLAG` enum: `DEFAULT_QA_STORY_MODE`.

## How do I test this PR?



Enable the `DEFAULT_QA_STORY_MODE` and `ENABLE_QA` feature flags and observe new stories being created with the Q&A story mode by default.
